scripts/rc-params.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
```

Comment:
nit: do we really need to have a separate script for these constants? If there are no reasons to have the separated, I'd rather move them into `download-rc.js`.